### PR TITLE
[release-v1.5] Add plugins: event, kafka, workflow

### PR DESCRIPTION
**Pull request by dsimansk**
/cc @rhuss @vyasgun

### CODE REVIEW

File: hack/build.sh.d/kn-plugin-func.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# There no midstream image currently, fallback to upstream one.
+readonly socat_image="${KN_PLUGIN_FUNC_SOCAT_IMAGE:-quay.io/boson/alpine-socat:1.7.4.3-r1-non-root}"
+export EXTERNAL_LD_FLAGS="${EXTERNAL_LD_FLAGS:-} \
+-X knative.dev/kn-plugin-func/k8s.SocatImage=${socat_image} "
```
Comment (dsimansk) [inline hack/build.sh.d/kn-plugin-func.sh:6]: That's a productization hook. It's fine to keep upstream image here as default. It's overriden during prod build.

File: hack/build.sh.d/kn-plugin-workflow.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+readonly quarkus_version="2.7.6.Final-redhat-00006"
+readonly quarkus_platform_group_id="com.redhat.quarkus.platform"
+export EXTERNAL_LD_FLAGS="${EXTERNAL_LD_FLAGS:-} \
+-X github.com/kiegroup/kie-tools/packages/kn-plugin-workflow/pkg/metadata.QuarkusVersion=${quarkus_version} \
+-X github.com/kiegroup/kie-tools/packages/kn-plugin-workflow/pkg/metadata.QuarkusPlatformGroupId=${quarkus_platform_group_id}"
```
Comment (dsimansk) [inline hack/build.sh.d/kn-plugin-workflow.sh:7]: @ljmotta I'll be tentatively online next week during KubeCon. And on the PTO afterwards. Please if you can make sure this version are updated for Serverless 1.26 release.

File: go.mod [hunk 3]
```diff
@@ -132,12 +132,46 @@ require (
...
+	github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21 // indirect
+	github.com/eapache/queue v1.1.0 // indirect
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/hashicorp/go-uuid v1.0.2 // indirect
+	github.com/jcmturner/aescts/v2 v2.0.0 // indirect
+	github.com/jcmturner/dnsutils/v2 v2.0.0 // indirect
+	github.com/jcmturner/gofork v1.0.0 // indirect
+	github.com/jcmturner/gokrb5/v8 v8.4.2 // indirect
+	github.com/jcmturner/rpc/v2 v2.0.3 // indirect
+	github.com/kiegroup/kie-tools/packages/kn-plugin-workflow v0.23.1
```
Comment (dsimansk) [inline go.mod:154]: @ljmotta and new version of plugin here. 

Manually setting the tag or using `go get -u` mechanism and then running (handles vendor, licenses etc.):
```
./hack/update-deps.sh
```

**Review by rhuss — COMMENTED** [review]
/approve
/lgtm

File: go.mod (hunk 3 above)
Comment (dsimansk) [reply go.mod:154]: @ederign FYI ^

File: hack/build.sh.d/kn-plugin-func.sh (hunk 1 above)
Comment (dsimansk) [reply hack/build.sh.d/kn-plugin-func.sh:6]: @ederign FYI ^

Comment (ederign) [reply hack/build.sh.d/kn-plugin-func.sh:6]: @dsimansk, thanks for the detailed instructions. As @ljmotta will be on PTO, @paulovmr or I will work on that.

Our team is still working with the Quarkus team to figure out what will be the specific quarkus version (probably Quarkus 2.14.0.Final ).

Just to double-check on dates and timings:

1-) The cut-off for this upgrade will be around November 8th ?
2-) As you are going to PTO, who should we reach out to review/approve the PR with the version changes?

Thanks!

File: go.mod (hunk 3 above)
Comment (ederign) [reply go.mod:154]: PR sent: https://github.com/openshift/knative-client/pull/1158

**Pull request merged**